toolkit/scripts/pkggen.mk

```
@@ -59,6 +59,7 @@ clean-spec-parse:
 	@echo Verifying no mountpoints present in $(parse_working_dir)
 	$(SCRIPTS_DIR)/safeunmount.sh "$(parse_working_dir)" && \
 	rm -rf $(parse_working_dir)
+	rm -rf $(specs_file)*
```

Nit-pick: do we need the `*`?
```suggestion
	rm -rf $(specs_file)
```